lib/components/ToasterDialog/ToasterDialog.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import { Meta, StoryFn } from '@storybook/react'
+
+import { ToastContainer } from 'react-toastify'
+
+import { default as ToasterDialogComponent } from './ToasterDialog'
+import { DialogProvider } from '../../context'
+import React from 'react'
+import Utils from '../../utils'
+import Button from '../Button'
+import { CloseButton } from '../index'
+
+import 'react-toastify/dist/ReactToastify.css'
+
+interface StoryArgs {
+  successMessage: string
+  errorMessage: string
+}
+
+interface toasterStoryArgs {
+  successToastMessage: string
+  errorToastMessage: string
+}
+
+const DialogStory = (args: StoryArgs) => (
+  <>
+    <div style={{ display: 'flex', gap: '20px' }}>
+      <Button onClick={() => Utils.toastSuccess(args.successMessage)}>
```

Comment:
let's pass here 'true' for showDialog since here we always want it to appear as Dialog, and not as Toaster, even when the message is small:
`onClick={() => Utils.toastSuccess(args.successMessage, true)}`